### ISSUE TITLE
ZdcPluginMissing: SSL/TLS plugin initailization failed (VM plugin missing ? OS libraries missing ?) on Debian Stretch

### ISSUE THREAD

**Issue by riverdusty** [GsDevKit/GsDevKit_home#224]
Followed instructions exactly and got this on `createStone devKit_33 3.3.0`

```
Fetched -> BaselineOfSton-dkh.5 --- filetree:///home/dusty/GsDevKit_home/shared/repos/ston/repository [:] --- filetree:///home/dusty/GsDevKit_home/shared/repos/ston/repository
Loaded -> BaselineOfSton-dkh.5 --- filetree:///home/dusty/GsDevKit_home/shared/repos/ston/repository [:] --- filetree:///home/dusty/GsDevKit_home/shared/repos/ston/repository
Fetched -> BaselineOfTode-dkh.125 --- filetree:///home/dusty/GsDevKit_home/shared/repos/tode/repository [:] --- filetree:///home/dusty/GsDevKit_home/shared/repos/tode/repository
Loaded -> BaselineOfTode-dkh.125 --- filetree:///home/dusty/GsDevKit_home/shared/repos/tode/repository [:] --- filetree:///home/dusty/GsDevKit_home/shared/repos/tode/repositoryioLoadModule(/home/dusty/GsDevKit_home/shared/pharo/pharo-vm/libSqueakSSL.so):
  libssl.so.1.0.0: cannot open shared object file: No such file or directory
'Errors in script loaded from /home/dusty/GsDevKit_home/sys/default/pharo/gsDevKitCommandLineLoad.st'
ZdcPluginMissing: SSL/TLS plugin initailization failed (VM plugin missing ? OS libraries missing ?)
ZdcPluginSSLSession>>initialize in Block: [ :exception | ...
BlockClosure>>cull:
MethodContext(ContextPart)>>handleSignal: in Block: [ self exceptionHandlerBlock cull: exception ]
BlockClosure>>ensure:
MethodContext(ContextPart)>>handleSignal:
PrimitiveFailed(Exception)>>signal
PrimitiveFailed class(SelectorException class)>>signalFor:
ZdcPluginSSLSession(Object)>>primitiveFailed:
ZdcPluginSSLSession(Object)>>primitiveFailed
ZdcPluginSSLSession>>primitiveSSLCreate
ZdcPluginSSLSession>>initialize in Block: [ handle := self primitiveSSLCreate ]
BlockClosure>>on:do:
ZdcPluginSSLSession>>initialize
ZdcPluginSSLSession class(Behavior)>>new
ZdcSecureSocketStream>>sslSession
ZdcSecureSocketStream>>connect
ZnNetworkingUtils>>socketStreamToUrlDirectly:
ZnNetworkingUtils>>socketStreamToUrl:
ZnNetworkingUtils class>>socketStreamToUrl:
ZnClient>>getConnectionAndExecute
ZnClient>>executeWithRedirectsRemaining:
ZnClient>>executeWithRetriesRemaining: in Block: [ self executeWithRedirectsRemaining: self maxNumb...etc...
BlockClosure>>on:do:
ZnClient>>executeWithRetriesRemaining:
ZnClient>>executeWithTimeout in Block: [ self executeWithRetriesRemaining: self numberOfR...etc...
BlockClosure>>on:do:
ZnClient>>executeWithTimeout in Block: [ ...
ZnClient>>withTimeoutDo: in Block: [ ^ block value ]
ZnConnectionTimeout(DynamicVariable)>>value:during: in Block: [ ...
BlockClosure>>ensure:
Error on or near line 142 :: create_gs_server_image -d /home/dusty/GsDevKit_home/shared/pharo gsDevKitCommandLine :: create_gs_server_image -d /home/dusty/GsDevKit_home/shared/pharo gsDevKitCommandLine
Error on or near line 74 :: devKitCommandLine createSmalltalkCIStoneNew devKit_33 3.3.0 DataCurator swordfish  :: devKitCommandLine createSmalltalkCIStoneNew devKit_33 3.3.0 DataCurator swordfish
Error on or near line 203 :: createStone devKit_33 3.3.0 :: createStone devKit_33 3.3.0
```

`Linux evilcorp 4.9.0-8-amd64 #1 SMP Debian 4.9.130-2 (2018-10-27) x86_64 GNU/Linux
`

**Comment by riverdusty** [comment]
looks like I need to manually install libssl.so.1.0.0.
Need to update libSqueakSSL methinks

**Comment by dalehenrich** [comment]
@riverdusty, did you run one of the `install*` before running createStone? ... If you have, the [installOsPrereqs](https://github.com/GsDevKit/GsDevKit_home/blob/master/bin/utils/installOsPrereqs) script may not recognize/work correctly with your version of Debian...

**Comment by dalehenrich** [comment]
[Here's the function that installs the prereqs](https://github.com/GsDevKit/GsDevKit_home/blob/master/bin/utils/installOsPrereqs#L84-L94)

**Comment by riverdusty** [comment]
You are correct, in that function it calls the installUbuntu function which attempts to install `    sudo apt-get -y install libssl1.0.0:i386` which Debian doesn't recognise. 
After installing (manually) the libssl1.0.2:i386 and doing ln -s to point to it, I got another error, wrong version.
So I deleted the entire Gs folder and started from scratch and it worked.
I will have to retest again, but if its simply a matter of having libssl1.0.2:i386 installed first, then we'd need to change the function for prereqs for Debian to install 1.0.2 first.

**Comment by dalehenrich** [comment]
I don't have a Debian system to test with ... if you could put together a PR with a script that works for Debian it would be much appreciated!

**Comment by riverdusty** [comment]
As soon as I get a chance I will do exactly that! Give me a week though

**Comment by dalehenrich** [comment]
Great!

**Comment by dalehenrich** [comment]
addressed with PR #242
